zone.ts

```diff
@@ -1,18 +1,52 @@
 export enum Zone {
-    INDAR = 2,
-    HOSSIN = 4,
-    AMERISH = 6,
-    ESAMIR = 8,
-    // VR_TRAINING_NC = 96,
-    // VR_TRAINING_TR = 97,
-    // VR_TRAINING_VS = 98,
-    OSHUR = 344
+  INDAR = 2,
+  HOSSIN = 4,
+  AMERISH = 6,
+  ESAMIR = 8,
+  // VR_TRAINING_NC = 96,
+  // VR_TRAINING_TR = 97,
+  // VR_TRAINING_VS = 98,
+  OSHUR = 344,
 }
 
-export const zoneArray = [
-    2,
-    4,
-    6,
-    8,
-    344,
-];
+export const zoneArray = [2, 4, 6, 8, 344]
+export const zoneToWarpgateArray = new Map<Zone, number[]>([
+  [Zone.INDAR, [7801, 120000, 4801]],
+  [Zone.HOSSIN, [308000, 309000, 310000]],
+  [Zone.AMERISH, [200000, 201000, 203000]],
+  [Zone.ESAMIR, [258000, 259000, 260000]],
+  [Zone.OSHUR, [400370, 400369, 400371]],
+])
+
+export const ZoneBgClassString = (zone: Zone | undefined): string => {
```

Comment:
Website specific

Reply:
But we have zone.ts from our submodule and then a seperate one for the website with the same name that will make it very confusing

Reply:
We shouldn't be putting a module's specific constants in a shared module. 

Also check my comments in the website PR, you'll find your answer in there

Reply:
I agree we shouldn't be putting module specific stuff into a shared place, however the only other way would either be 
1. Make a new constants file called something different and re-route specifically those files that are looking for specifically this object to go to it.
2. Or make a new file with the same name and then re-route the ones that are looking for the object to this but in this case we know have two files in the project with the same name. Say if someone was to update this file later and they wanted to use this object again they would have to work out where it is stored in, this also applies in the 1. scenario.
So I just think that this is the lesser of the two evils.